python/run-tests.py

```diff
@@ -82,7 +82,7 @@ def prepare(root_dir):
     sbt_path = "sbt"
     delete_if_exists(os.path.expanduser("~/.ivy2/cache/com/microsoft/hyperspace"))
     delete_if_exists(os.path.expanduser("~/.m2/repository/com/microsoft/hyperspace"))
-    run_cmd([sbt_path, "clean", "publishM2"], stream_output=True)
+    run_cmd([sbt_path, "project spark2_4", "clean", "publishM2"], stream_output=True)
```

Comment:
We only run python tests with Spark 2.4. There is no need to build other versions.

Reply:
Split to a seperate PR